### PR TITLE
chore: migrate to Python 3.11

### PR DESCRIPTION
**Pull request by raphael0202**
### What

Upgrade to Python 3.11.

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -13,7 +13,6 @@ theme:
 
 markdown_extensions:
   - footnotes
-  - mdx_truly_sane_lists
```
[the remark this answers is not in the document]
Comment (alexgarel) [reply mkdocs.yml:16]: see https://github.com/Python-Markdown/markdown/issues/1204#issuecomment-992499686

[the remark this answers is not in the document]
Comment (raphael0202) [reply mkdocs.yml:16]: Indeed I removed it because it's not compatible with Python3.11

[the remark this answers is not in the document]
Comment (raphael0202) [reply mkdocs.yml:16]: I pinned markdown version (as the original repo didn't) and it works as expected.
I opened a PR on the original repo there: https://github.com/Python-Markdown/markdown/blob/master/docs/change_log/release-3.4.md

[the remark this answers is not in the document]
Comment (alexgarel) [reply mkdocs.yml:16]: interesting, thanks! (Is your link about the PR right?)

[the remark this answers is not in the document]
Comment (raphael0202) [reply mkdocs.yml:16]: No indeed wrong link ;).
https://github.com/radude/mdx_truly_sane_lists/pull/20

**Pull request merged**